python/lsst/obs/base/tests.py

```diff
@@ -80,6 +82,7 @@ def setUp_tests(self, butler, mapper, dataIds):
         self.butler = butler
         self.mapper = mapper
         self.dataIds = dataIds
+        self.log = Log.getLogger('ObsTests')
```

Comment:
Do we use the logger anywhere in these tests?

Reply:
We do now...